nncf/tensorflow/graph/metatypes/common.py

```diff
@@ -51,14 +56,20 @@
     layer_metatypes.TFDepthwiseConv2DLayerMetatype,
     layer_metatypes.TFConv1DTransposeLayerMetatype,
     layer_metatypes.TFConv2DTransposeLayerMetatype,
-    layer_metatypes.TFConv3DTransposeLayerMetatype
+    layer_metatypes.TFConv3DTransposeLayerMetatype,
+    # GENERAL_CONV_TF_OP_METATYPES
```

Comment:
Does this refer to a certain identifier for the container (list, dict) anywhere? If so, perhaps the asterisk-expansion syntax would be more convenient (i.e. `*GENERAL_CONV_TF_OP_METATYPES`).

Reply:
I use this comment as the delimiter to divide the `layer_metatypes` from the `op_metatypes`.